deploy-agent/tox.ini

```diff
@@ -14,4 +14,4 @@ commands=py.test --cov=deployd {posargs}
 python =
     3.6: py36
```

Comment:
earlier, we have py37, py38? should they be consistent here

Reply:
Updated the versions in the GitHub workflow for consistency